packages/datastore/src/schema.ts

```diff
@@ -46,7 +46,7 @@ function validateSchema(schema: Schema): string[] {
   const errors = [];
   // Ensure that field names do not begin with `$` or `@`.
   for (let name in schema.fields) {
-    if (invalidFielnameLeads.indexOf(name[0]) !== -1) {
+    if (invalidFieldnameLeads.indexOf(name[0]) !== -1) {
```

Comment:
why `indexOf` and not simply `===`?

Reply:
Not sure, I was just correcting a typo here.